homeassistant/components/backup/config.py

```diff
@@ -176,14 +176,8 @@ async def update(
                         self.data.agents[agent_id], **agent_config
                     )
         if create_backup is not UNDEFINED:
-            if (new_agent_ids := create_backup.get("agent_ids", [])) and (
-                discarded_agent_ids := set(self.data.create_backup.agent_ids)
-                - set(new_agent_ids)
-            ):
-                for agent_id in discarded_agent_ids:
-                    delete_automatic_backup_agents_unavailable_issue(
-                        self._hass, agent_id
-                    )
+            if new_agent_ids := create_backup.get("agent_ids"):
+                check_unavailable_agents(self._hass, self._manager, new_agent_ids)
```

Comment:
This means we won't update issues if agent_ids is empty, I don't think that's correct, we should change to:
```suggestion
            if "agent_ids" in create_backup:
                check_unavailable_agents(self._hass, self._manager, create_backup["agent_ids"])
```